connman/src/main.c

```diff
@@ -815,6 +858,17 @@ bool connman_setting_get_bool(const char *key)
 	return false;
 }
 
+unsigned int connman_setting_get_uint(const char *key)
+{
+	if (g_str_equal(key, CONF_ONLINE_CHECK_INITIAL_INTERVAL))
```

Comment:
g_str_equal() does not handle null args, do we know that "key" is always non-null?

Reply:
I think this is good since the developer would instantly see that `NULL` keys are causing instant crashes. Kind of ensures that code remains correct, as searching for uses of `connman_settings_get_` shows. Lets leave as is.